ivy/functional/frontends/paddle/vision/transforms.py

```diff
@@ -13,3 +13,8 @@
 def to_tensor(pic, data_format="CHW"):
     array = ivy.array(pic)
     return Tensor(array)
+
+@to_ivy_arrays_and_back
+def vflip(img):
+  x= ivy.array(img)
+  return ivy.flip(x,axis=0)
```

Comment:
Thanks for the PR, how do you feel about this suggestion? 

```suggestion
  return ivy.flip(ivy.array(img),axis=0)
```

Note: I am not an ivy member.

Reply:
it's great and minimum line. I am happy to commit your suggestion.

Reply:
You are welcome.